pytorch_lightning/trainer/data_loading.py

```diff
@@ -63,14 +69,11 @@ def init_train_dataloader(self, model):
             self.val_check_batch = self.val_check_interval
             if self.val_check_batch > self.num_training_batches:
                 raise ValueError(
-                    f"val_check_interval ({self.val_check_interval}) must be less than or equal to "
-                    f"the number of the training batches ({self.num_training_batches}). "
-                    f"If you want to disable validation set val_percent_check to 0.0 instead.")
+                    f"`val_check_interval` ({self.val_check_interval}) must be less than or equal "
```

Comment:
pls pep8 prefers to have whitespace on the beginning of next line instead of the last line ending

Reply:
@kuynzereb have you check this?

Reply:
Sorry, I haven't fixed it yet and PR has been already merged. But I got your point. From now on I will not use trailing spaces in multiline strings :)

Reply:
well if you feel like nothing to do, you may check all string in the package... ;]